### PR TITLE
Update ci:packages command and fix figure bug

### PR DESCRIPTION
**Pull request by dr3**
Required for https://github.com/bbc/psammead/pull/290

**Overall change:** Doesnt force using local packages when running `npm run ci:packages` which allows for tests to be run at their published state, not what is available locally.

**Code changes:**

- Update commands to not force local dependancies and support ci and local
- Bumps psammead-fixtures dependancies to fix SC issue.

**Commands:**
- `ci:packages` - `"npm ci && lerna clean --yes && lerna bootstrap --ci --hoist --nohoist=@bbc/*"`
For use on ci

Why `--hoist`? This is to avoid an issue where styled components needs to be a singleton and if it isnt, doesnt render styles in snapshots

Why `--nohoist=@bbc/*`? Without this you get issues like `Missing: @bbc/gel-foundations@^0.1.2` where it for some reason cant find the version of something that is available on npm

---
- `install:packages` - `"npm i && lerna clean --yes && lerna bootstrap --hoist"`
For use locally

Why no `--nohoist=@bbc/*`? If you use nohoist you get an error `npm ERR! Cannot read property 'match' of undefined` which is to do with a corrupted package-lock. Unknow why this happens.

---
- `install:packages:link` - `"npm i && lerna clean --yes && lerna bootstrap --hoist --force-local"`
For use locally when developing linked packages

Why `lerna clean --yes`? This is because for some reason, if you try to install locally without linking, after linking, you get the error `Missing write access to /Users/mcmild07/dev/psammead/packages/components/psammead-figure/node_modules/@bbc/gel-foundations` which is where it tries to write into a folder that is a sym link

TL;DR Lerna is gross https://github.com/bbc/psammead/issues/297
---

Testing notes; The above commands should work as expected locally in any order. Regression only.

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: packages/components/psammead-figure/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 | Version | Description |
 |---------|-------------|
+| 0.1.3   | [PR#295](https://github.com/BBC/psammead/pull/295) Update dependances. |
```
Comment (bcmn) [inline packages/components/psammead-figure/CHANGELOG.md]:
```suggestion
| 0.1.3   | [PR#295](https://github.com/BBC/psammead/pull/295) Update dependancies. |
```

😇

File: README.md [hunk 2]
```diff
@@ -43,6 +51,8 @@ npm test
 
 This runs Jest across any packages matching this glob pattern: `packages/components/**/*.test.jsx`.
 
+NB: You cant run the jest tests when the packages are linked locally. Please run `npm run install:packages` or `npm run ci:packages` to reset all links.
```
Comment (bcmn) [inline README.md]:
```suggestion
NB: You can't reliably run the jest tests when the packages are linked locally, as they may have been linked across breaking changes. Please run `npm run install:packages` or `npm run ci:packages` to reset all links.
```

**Review by bcmn — COMMENTED** [review]
Looks good overall -- just one suggestion to clarify the line in the readme.

**Review by bcmn — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -5,10 +5,11 @@
   "main": "index.js",
   "scripts": {
     "build": "lerna exec --parallel -- babel src -d dist --ignore '**/*.test.jsx,**/*.stories.jsx' --root-mode upward",
-    "ci:packages": "npm ci && lerna bootstrap --ci --hoist --force-local",
+    "ci:packages": "npm ci && lerna clean --yes && lerna bootstrap --ci --hoist --nohoist=@bbc/*",
```
Comment (sareh) [inline package.json:8]: Instead of having `--hoist --nohoist=@bbc/*` could we explicitly just hoist styled-components?
e.g. `--hoist=styled-components`?

Comment (dr3) [reply package.json:8]: I have tried doing this, however lerna seems flakey when asking to hoist a specific package. It always misses a SC instance. Tried many different hoisting options with no success

**Review by sareh — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (sareh) [reply package.json:8]: Thanks for looking into this, Drew.

**Comment by dr3** [comment]
https://gist.github.com/dr3/654b38a8487f7e6beec7f7a47a5684f9#file-1_fresh_then_ci-packages_then_test-unit-txt
Running `ci:packages` after a fresh clone

---

https://gist.github.com/dr3/654b38a8487f7e6beec7f7a47a5684f9#file-2_install-packages_then_test-unit-txt
Running `test:unit` after `ci:packages` 

---

https://gist.github.com/dr3/654b38a8487f7e6beec7f7a47a5684f9#file-3_install-packages-link_then_test-unit-txt
Running `install:packages` after `test:unit` 


---

https://gist.github.com/dr3/654b38a8487f7e6beec7f7a47a5684f9#file-4_makechange_then_build-txt
This shows making a change with a breaking change and building

---

https://gist.github.com/dr3/654b38a8487f7e6beec7f7a47a5684f9#file-5_test-unit_fail_then_install-packages_then_test-unit_pass-txt
Shows how after doing this, when installed with links, the tests fail. BUT after running `install:packages`, which doesnt link, it passes. This is the desired behaviour.

**Comment by jamesbrumpton** [comment]
👍
